v2.6/getting-started/kubernetes/upgrade/convert.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Converting policies
+redirect_from: latest/getting-started/kubernetes/upgrade/convert
+---
+
+If you are upgrading from the following, you must convert your policies
+before you can upgrade the Calico components.
+
+| Current version     | Datastore type           | Conversion required? |
+| ------------------- | ------------------------ | -------------------- |
+| earlier than v2.3.0 | Kubernetes API datastore | Yes                  |
+| earlier than v2.4.0 | etcd                     | Yes                  |
+
+Calico's interpretations of policy [changed]((https://github.com/kubernetes/kubernetes/pull/39164#issue-197243974)) 
+after this. To maintain behavior when upgrading, you should follow these steps 
+prior to upgrading Calico to ensure your configured policy is
+enforced consistently throughout the upgrade process.
+
+- **In any namespace that has a "DefaultDeny" annotation**
+
+  Delete any `NetworkPolicy` objects in that namespace.  After upgrade, these 
```

Comment:
This is backwards - any namespace that has the annotation should get the new policy created.

Any policy without it should have its policies deleted.